bs-react-native-next/src/apis/PanResponder.re

```diff
@@ -0,0 +1,76 @@
+type gestureState;
+
+[@bs.obj]
+external gestureState:
+	(
+		~stateID: float,
+		~moveX: float,
+		~moveY: float,
+		~x0: float,
+		~y0: float,
+		~dx: float,
+		~dy: float,
+		~vx: float,
+		~vy: float,
+		~numberActiveTouches: int
+	) =>
+	gestureState =
+	"";
+
+[@bs.get] external stateID: gestureState => float = "";
+[@bs.get] external moveX: gestureState => float = "";
+[@bs.get] external moveY: gestureState => float = "";
+[@bs.get] external x0: gestureState => float = "";
+[@bs.get] external y0: gestureState => float = "";
+[@bs.get] external dx: gestureState => float = "";
+[@bs.get] external dy: gestureState => float = "";
+[@bs.get] external vx: gestureState => float = "";
+[@bs.get] external vy: gestureState => float = "";
+[@bs.get] external numberActiveTouches: gestureState => int = "";
+
+type config;
+
+[@bs.obj]
+external config: 
+	(
+		~onMoveShouldSetPanResponder: (Event.t, gestureState) => bool=?,
+		~onMoveShouldSetPanResponderCapture: (Event.t, gestureState) => bool=?,
+		~onStartShouldSetPanResponder: (Event.t, gestureState) => bool=?,
+		~onStartShouldSetPanResponderCapture: (Event.t, gestureState) => bool=?,
+		~onPanResponderReject: (Event.t, gestureState) => unit=?,
+		~onPanResponderGrant: (Event.t, gestureState) => unit=?,
+		~onPanResponderStart: (Event.t, gestureState) => unit=?,
+		~onPanResponderEnd: (Event.t, gestureState) => unit=?,
+		~onPanResponderRelease: (Event.t, gestureState) => unit=?,
+		~onPanResponderMove: (Event.t, gestureState) => unit=?,
+		~onPanResponderTerminate: (Event.t, gestureState) => unit =?,
+		~onPanResponderTerminationRequest: (Event.t, gestureState) => bool=?,
+		~onShouldBlockNativeResponder: (Event.t, gestureState) => bool=?,
+		unit
+	) =>
+	config =
+	"";
+
+type t;
+
+[@bs.module "react-native"] [@bs.scope "PanResponder"]
+external create: config => t = "";
+
+type panHandlers;
+
+[@bs.get] external panHandlers: t => panHandlers = "";
+
+type callback('a) = Event.t => 'a;
+
+[@bs.get ] external onMoveShouldSetResponder: panHandlers => callback(bool) = "";
```

Comment:
The code is not formatted using `refmt`, even though we have a pre-commit hook that should be doing that.

Reply:
@sgny what are you using to commit? Current repo should automatically setup git hooks as soon as you run "npm install" or "yarn", transparently (so refmt should be run when you commit)

Reply:
Sorry I've been using my scripts, but I will update those.